pkg/telemetry/prometheus/packets.go

```diff
@@ -44,6 +47,9 @@ var (
 	forwardLatency             atomic.Uint32
 	forwardJitter              atomic.Uint32
 
+	// Custom
+	promAsnBytes *prometheus.CounterVec
```

Comment:
how about latency and participant count per ASN?

Reply:
As discussed,  will be added later and in another PR.

Reply:
I don't recalling discussing about adding it in another PR. What's wrong with adding it in this PR?